src/puppetlabs/trapperkeeper/services/status/status_core.clj

```diff
@@ -84,7 +84,8 @@
 
 (def GcStatsV1
   {schema/Str {:count schema/Int
-               :total-time-ms schema/Int}})
+               :total-time-ms schema/Int
+               (schema/optional-key :last-gc-info) {:duration-ms schema/Int}}})
```

Comment:
I assume since we're only adding an optional field (but not removing anything) we can continue to consider this part of the v1 schema

Reply:
It's added as optional as it is possible for last-gc-info to be `nil` if no collection has occured yet. For example, with Puppet Server, the Major GC algorithm usually doesn't have to fire off a major collection sweep until an hour or so of uptime.

Reply:
Also, `last-gc-info` could be specific to Oracle's HotSpot JVM as it's not part of the GarbageCollectorMXBean interface. That's what PE uses, but I also flagged it as optional just in case someone is running Open Source Puppet Server or PuppetDB on a different JVM.